protected/humhub/modules/file/resources/js/humhub.file.js

```diff
@@ -404,7 +404,7 @@ humhub.module('file', function (module, require, $) {
             that.delete(file);
         });
 
-        if(!(this.isImage(file) && this.options.hideImageFileInfo)) {
+        if(!(this.options.hideImageFileInfo)) {
```

Comment:
This will just hide all files, we need somthing like the `isImage()`, but for all types of files we want to hide. Maybe we could also add this information in the file response.